video/out/drm_common.c

```diff
@@ -1075,10 +1030,7 @@ bool vo_drm_init(struct vo *vo)
 
     drm->ev.version = DRM_EVENT_CONTEXT_VERSION;
     drm->ev.page_flip_handler = &drm_pflip_cb;
-
-    drm->vsync_info.vsync_duration = 0;
-    drm->vsync_info.skipped_vsyncs = -1;
-    drm->vsync_info.last_queue_display_time = -1;
+    drm->present = mp_present_initialize(drm, 8); // max swapchain depth allowed
```

Comment:
If this is fixed size, what is the gain of using linked list? Why not use array to avoid constant pointer hopping to swap first/last element?

Reply:
Sure, whatever you prefer, it was just a lose idea.

Reply:
If the frames remain ordered a circular buffer should work.